packages/create-svelte/bin.js

```diff
@@ -75,8 +76,14 @@ async function main() {
 					message: 'Add type checking?',
 					initial: false,
 					choices: [
-						{ title: 'Type-checked JavaScript', value: 'checkjs' },
-						{ title: 'TypeScript', value: 'typescript' },
+						{
+							title: 'With JavaScript comments (JSDoc)',
```

Comment:
might need a bit more explanation of what this is. but hopefully this isn't too long
```suggestion
							title: 'Runs TypeScript's type checking against JavaScript comments (JSDoc)',
```

Reply:
Guess it could be put into the descriptions, how's this? 

![image](https://user-images.githubusercontent.com/50981692/174713045-4c6adfc5-0699-4b15-89c1-6532e9e89675.png)
![image](https://user-images.githubusercontent.com/50981692/174713051-bef5f516-09fe-4d3a-8545-b069386b8a93.png)

Reply:
I agree it looks nicer on the same line

Reply:
That's coming from how the prompts library handles descriptions, if they're too long to fit on one line they start from the next, so it has to be put in the title

![image](https://user-images.githubusercontent.com/50981692/174778288-15b09683-4255-433d-bd32-0e5ea1a39d0e.png)

Reply:
```
Type-Checked JavaScript - check for types in JS with JSDoc.
TypeScript - check for types with TS.
None - Don't check for types.

Reply:
Or how it was in https://github.com/sveltejs/kit/pull/5221#discussion_r902139470 but shortcut JS and TS, and maybe replace last part with `and run TSC to check types.`

Reply:
Turns out we can cheat a bit with kleur to make parts of the title look like the description

![image](https://user-images.githubusercontent.com/50981692/174787257-b9d70042-4c0f-47a9-83a6-243bf41b4e8e.png)

Ended up with this, which I think looks nice and readable.